include/ck_tile/ops/layernorm2d/pipeline/layernorm2d_fwd_pipeline_one_pass.hpp

```diff
@@ -111,8 +111,9 @@ struct Layernorm2dFwdPipelineOnePass
         {
             sweep_tile(x_resi, [&](auto idx) {
                 // compute x = x_resi + x
-                x(idx) = type_convert<YResidualDataType>(x_resi(idx)) +
-                         type_convert<YResidualDataType>(x(idx));
+                auto re_ = type_convert<ComputeDataType>(x_resi(idx)) +
```

Comment:
auto x = type_convert<ComputeDataType>(x_resi(idx)) + type_convert<ComputeDataType>(x(idx));
store_tile(y_residual_window, cast_tile<XDataType>(x));

Reply:
sure, will create another PR for this~